portal_recaptcha/views/portal_mixin.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2022 Coop IT Easy SC
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <template
+        id="assets_editor_recaptcha"
+        name="Google reCAPTCHA"
+        inherit_id="portal.frontend_layout"
+    >
+        <xpath expr="//main" position="inside">
+            <script src="https://www.google.com/recaptcha/api.js" />
```

Comment:
Why loading reCaptcha JS in all the pages? I also don't see where is placed for the specific login page.

Reply:
Thank you for the review. 

>Why loading reCaptcha JS in all the pages?

I fixed this, now the JS is only called when the widget template is called. 

>I also don't see where is placed for the specific login page.

I don't understand what you mean  Could you explain ?